andela/casablanca-rc
# #157311107 Implement digital wallet

#### What does this PR do?
Add the implementation of a digital wallet to the project.
#### Description of Task to be completed?
Users should be able to fund and pay with a digital wallet.
#### How should this be manually tested?
- clone the repo and change into the project directory
- start the application(instructions in the readme)
- visit the application in the browser 
- login/signup as a user
- visit your profile
- there should be a wallet section that displays your current balance and also an option to fund the wallet
- using the admin access, enable wallet payment in the payment dashboard.
- select some product to purchase
- in the checkout page, you should see an option to pay with your wallet 
#### What are the relevant pivotal tracker stories?
#157311107
#### Background Context
The transfer option shown in the profile page is not functional
#### Screenshots
<img width="1007" alt="screen shot 2018-05-11 at 10 35 20 am" src="https://user-images.githubusercontent.com/26048536/39917990-13321082-5507-11e8-896f-294ffc096543.png">   

![screencapture-localhost-3000-tag-shop-2018-05-11-10_44_29](https://user-images.githubusercontent.com/26048536/39918470-71758754-5508-11e8-88d7-65ef196bc929.png)

![screencapture-localhost-3000-cart-checkout-2018-05-11-10_36_27](https://user-images.githubusercontent.com/26048536/39918335-0d8254ac-5508-11e8-9fb1-2a25c7fa2200.png)
